client.go

```diff
@@ -244,6 +258,24 @@ func IgnoreServers(s []net.IP) func(*Client) error {
 	}
 }
 
+// IgnoreServers is a function option that allow you to pass an array s of IP's
+// which the client will permit when the DHCP Server respond with a
+// dhcp4.OptionServerIdentifier (Code 54) in this IP array.
+func PermitServers(s []net.IP) func(*Client) error {
+	return func(c *Client) error {
+		c.permitServers = s
+		return nil
+	}
+}
+
+// set tolerateMode
+func TolerateMode(mode TolerateModeType) func(*Client) error {
```

Comment:
Not sure if Tolerate Mode is a great name (Can't think of a better one so a little more doc would be great).